lib/jormungandr/src/Cardano/Wallet/Jormungandr/Network.hs

```diff
@@ -168,6 +171,9 @@ data JormungandrLayer m = JormungandrLayer
     , getInitialFeePolicy
         :: Hash "Genesis"
         -> ExceptT ErrGetInitialFeePolicy m FeePolicy
+    , getInitialSlotDuration
+        :: Hash "Genesis"
+        -> ExceptT ErrGetInitialSlotDuration m SlotLength
```

Comment:
What about also having here a `getInitialBlockchainParameters` instead of both `getInitialFeePolicy` and `getInitialSlotDuration`, with regards to my previous comment about `BlockchainParameters`

Reply:
done